tests/ui/needless_return.fixed

```diff
@@ -84,14 +84,14 @@ fn test_macro_call() -> i32 {
 }
 
 fn test_void_fun() {
-    //~^^ needless_return
+    //~^ needless_return
```

Comment:
Had to change these because the lint span now starts on the immediate previous line, not the one before